index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Global school home's page</title>
+ <script src="https://kit.fontawesome.com/84967187a9.js" crossorigin="anonymous"></script>
+ <link rel="stylesheet" href="css/mobile_style.css">
+ <link rel="stylesheet" href="css/Destop_style.css">
+</head>
+<body>
+ <header>
+  <div class="logo">
+   <a href=""><span>GLO</span>BAL SCH<span>OOL</span></a>
+  </div>
+  <nav>
+   <ul>
+    <li><a href="index.html">Home</a></li>
+    <li><a href="about.html">About</a></li>
+    <li><a href="#">Join</a></li>
+    <li><a href="#">Sponsor</a></li>
+    <li><a href="#">News</a></li>
+   </ul>
+   <div class="menu_bar">
+    <i class="fas fa-bars"></i>
+   </div>
+  </nav>
+ </header>
+ <main>
+  <section class="first_sec">
+   <div>
+    <p class="wel">"Welcome to GLOBAL SCHOOL"</p>
+    <h1 class="title">LET CONTRIBUTE TO THE DEVELOPPEMENT OF OUR COMMUNITY</h1>
+    <p class="intro">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Adipisci porro, quasi doloremque 
+     voluptate rerum tenetur iure, voluptatem consequuntur non quaerat iste delectus! Magnam hic quidem eius assumenda, at eos tenetur. </p>
+    <p class="hrs">2022.02.14(Mon) ~ 18(FRI)</p>
+    <p class="lieu">@ Greater Accra, Tersano police station</p>
+   </div>
+  </section>
+  <section class="snd_section">
+   <h2 class="Mpr">Main Program</h2><span class="dash"></span>
+   <div class="contain_program">
+    <div class="program">
+     <img src="images/lecture.png" alt="lecture's icon">
+     <h3>Lecture</h3>
+     <p>Listen to the speakers from various countries talk about there journey intotech.</p>
+    </div>
```

Comment:
- [ ] For this `Main Program` section please make sure you stick the design as much as possible by using the given `colors` #272A31, #EC5242, #D3D3D3; `font-family`: cocogoose and lato;
Take a look again below images 👇 
- Original design 👇 
![Screenshot (182)](https://user-images.githubusercontent.com/67087664/153584998-4792ee18-ff8a-4fc6-8980-ef0e02403794.png)

- Your design 👇 
![Screenshot (183)](https://user-images.githubusercontent.com/67087664/153584947-5ab6335d-eda6-42c8-adfc-99ada7137330.png)